tests/vues/contenu.js

```diff
@@ -0,0 +1,57 @@
+/* global Event */
+
+import { VueContenu } from '../../src/vues/contenu.js';
+import { unContenantVrac } from '../aides/contenant.js';
+import jsdom from 'jsdom-global';
+
+describe('vue contenu', function () {
+  let vue;
+  let pointInsertion;
+
+  beforeEach(function () {
+    jsdom('<div id="stock"></div>');
+    pointInsertion = document.getElementById('stock');
+    vue = new VueContenu(pointInsertion);
+  });
+
+  function checkVisibilite (expectVisible, contenu) {
+    if (expectVisible) {
+      expect(contenu.classList).to.not.contain('invisible');
+    } else {
+      expect(contenu.classList).to.contain('invisible');
+    }
+  }
+
+  it("initialise un contenu invisible tant que le contenant n'est pas ouvert", function () {
+    const element = document.getElementById('contenu');
+    expect(element.classList.value).to.contain('invisible');
+  });
+
+  it('sais se cacher', function () {
+    vue.affiche(unContenantVrac('Nova Sky', 1));
+    const element = document.getElementById('contenu');
+    element.dispatchEvent(new Event('click'));
```

Comment:
Je trouve quand même cette écriture plus lourde que `$('#contenu').click()`… 😓

Reply:
`expect($('.contenu').hasClass('invisible')).to.be(true)` ?

Reply:
Ce qui m'intéresse dans la forme `expect(element.classList).to.contain('invisible');`, c'est qu'en cas d'échec, j'ai un message du type "invisible" n'est pas présent dans la liste "…, …, …"

Reply:
Ok. Restons-en là.